### ISSUE TITLE
Init via ajax

### ISSUE THREAD

**Issue by sergeizhukov** [Simonwep/pickr#147]
bug

#### What is the current behavior?

I have an ajax request with (this is came as ajax response):
```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@simonwep/pickr/dist/themes/classic.min.css"/> <!-- 'classic' theme -->
 <script src="https://cdn.jsdelivr.net/npm/@simonwep/pickr/dist/pickr.es5.min.js"></script>

<script>
     let colorPickerConf = {}; //common configuration from docs
     const colorPickr = Pickr.create(colorPickerConf);

</script>
```
After the first ajax request i see in console:
![image](https://user-images.githubusercontent.com/44158607/64114420-31bc0800-cd8d-11e9-8bc7-f239f5d4fc80.png)

But, if I do the second request (the same). It works properly. Is it correct?

UPD:
So, I carry out links **classic.min.css** and **pickr.es5.min.js** out of AJAX in `<head>` section. So now it works but: The pickr ignore the _container_ property! And automatically attaches on body tag
#### If the current behavior is a bug, please provide the steps to reproduce and create a [JSFiddle](https://jsfiddle.net/Simonwep/qx2Lod6r/).

cant reproduce on fiddle
#### Your environment:
```
Version (see Pickr.version): https://cdn.jsdelivr.net/npm/@simonwep/pickr/dist/pickr.es5.min.js
Used bundle (es5 or normal one): both
Used theme (default is classic): classic
Browser-version:  actual
Operating-system:  macOS
```

**Comment by Simonwep** [comment]
It seems like the scripts get loaded asynchronously, be sure to [not load it with a defer attribute](https://github.com/Simonwep/pickr#browser). And it can be reproduced, just used the code you posted: https://jsfiddle.net/Simonwep/c985sh6p

Unfortunatly the README is always up-to-date with the latest branch ([see this note](https://github.com/Simonwep/pickr#node)). To be sure to see the docs for you _currently used version_ please pick a revision (in your case [1.4.0](https://github.com/Simonwep/pickr/tree/4f911097eb9cd0260887c7bb74d7fc9ca61e19f8)).

The `container` option is pretty new and gets released with v1.4.1 in the next days (or maybe later today).

**Comment by sergeizhukov** [comment]
Thank you for the answer. I used the links from cdn so seems it was old version. With a new version the container property works now properly. I set async false, before for ajax request make. Idk how can be defer.
